### PR TITLE
Created basic base template

### PR DESCRIPTION
**Pull request by patrickkenyon**
Very loosely used the following as inspiration: http://beetox.ru/

Tested in latest Chrome and Safari browsers as well as Explorer 10 running on Windows 7.

style.css and portfolio.html need to be reviewed.

All help and feedback welcome!

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -4,10 +4,99 @@
     <meta charset="UTF-8">
     <title>portfolio</title>
     <link rel="stylesheet" href="style.css">
```
Comment (imogenhallett) [inline portfolio.html]: This should come after normalize.css

File: portfolio.html [hunk 2]
```diff
@@ -4,10 +4,99 @@
...
+    <link rel="normalize" type="text/css>" href="normalize.css">
```
Comment (imogenhallett) [inline portfolio.html]: type="text/css>" is incorrect.  Should be type="text/css"> note the position of the closing "

File: portfolio.html [hunk 3]
```diff
@@ -4,10 +4,99 @@
...
 </head>
+
```
Comment (imogenhallett) [inline portfolio.html]: Consider removing this whitespace

File: portfolio.html [hunk 4]
```diff
@@ -4,10 +4,99 @@
...
 <body>
+
     <header>
```
Comment (imogenhallett) [inline portfolio.html]: Consider removing this white space

File: portfolio.html [hunk 5]
```diff
@@ -4,10 +4,99 @@
...
 
+        <nav class="main-nav">
+            <a href="">Home</a>
```
Comment (imogenhallett) [inline portfolio.html]: This link has no destination

File: portfolio.html [hunk 6]
```diff
@@ -4,10 +4,99 @@
...
+            <a href="#aboutme">About</a>
+            <a href="#portfolio">Portfolio</a>
+            <a href="#contact">Contact info</a>
+        </nav>
     </header>
+
+    <section>
```
Comment (imogenhallett) [inline portfolio.html]: Consider putting the class mini-profile on the section rather than creating a div

File: style.css [hunk 7]
```diff
@@ -1,5 +1,190 @@
...
+    color: #fff;
+    background-image: url("https://placekitten.com/700/300");
+    background-size: cover;
+    min-height: 300px;
+    background-position: center;
+    position: relative;
+    padding: 0 100px 0 100px;
 }
 
+@media only screen and (max-width: 600px) {
```
Comment (imogenhallett) [inline style.css]: You already have a media query for max-width: 600px - see line 43

File: style.css [hunk 8]
```diff
@@ -1,5 +1,190 @@
...
+}
+
+.about-me-holder:after {
+    width: 100%;
+    clear: both;
+    display: block;
+    content: ' ';
+}
+
+@media only screen and (max-width: 600px) {
```
Comment (imogenhallett) [inline style.css]: You already have a media query for wax-width 600px - see line 43

File: style.css [hunk 9]
```diff
@@ -1,5 +1,190 @@
...
+@media only screen and (max-width: 845px) {
+    .about-me {
+        margin: auto;
+        float: none;
+        text-align: center;
+        padding: 0 4px 0 4px;
+    }
+}
 
+@media only screen and (max-width: 845px) {
```
Comment (imogenhallett) [inline style.css]: You already have a media query for max-width 845px - see line 175

**Review by imogenhallett — COMMENTED** [review]
Just a few things - looks good

File: portfolio.html (hunk 1 above)
Comment (patrickkenyon) [reply portfolio.html]: Changed

File: portfolio.html (hunk 2 above)
Comment (patrickkenyon) [reply portfolio.html]: Changed

File: portfolio.html (hunk 3 above)
Comment (patrickkenyon) [reply portfolio.html]: Changed

File: portfolio.html (hunk 4 above)
Comment (patrickkenyon) [reply portfolio.html]: Changed

File: portfolio.html (hunk 5 above)
Comment (patrickkenyon) [reply portfolio.html]: Removed

File: portfolio.html (hunk 6 above)
Comment (patrickkenyon) [reply portfolio.html]: Changed

File: style.css (hunk 7 above)
Comment (patrickkenyon) [reply style.css]: Consolidated media queries and placed them at the end of the doc.

File: style.css (hunk 8 above)
Comment (patrickkenyon) [reply style.css]: Consolidated media queries and placed them at the end of the doc.

File: style.css (hunk 9 above)
Comment (patrickkenyon) [reply style.css]: Consolidated media queries and placed them at the end of the doc.

File: portfolio.html [hunk 10]
```diff
@@ -1,13 +1,96 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="normalize" type="text/css" href="normalize.css">
     <meta charset="UTF-8">
-    <title>portfolio</title>
```
Comment (imogenhallett) [inline portfolio.html:5]: Duplicate title tag - please remove

File: portfolio.html [hunk 11]
```diff
@@ -1,13 +1,96 @@
...
+    <title>Patrick| Portfolio</title>
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
     <header>
-
```
Comment (imogenhallett) [inline portfolio.html:10]: White space please remove

File: portfolio.html (hunk 10 above)
Comment (patrickkenyon) [reply portfolio.html:5]: Changed

File: portfolio.html (hunk 11 above)
Comment (patrickkenyon) [reply portfolio.html:10]: Changed

File: portfolio.html (hunk 10 above)
Comment (patrickkenyon) [inline portfolio.html:5]: Changed

File: portfolio.html [hunk 12]
```diff
@@ -1,13 +1,96 @@
...
+        <nav class="main-nav">
+            <a href="#aboutme">About</a>
+            <a href="#portfolio">Portfolio</a>
+            <a href="#contact">Contact info</a>
+        </nav>
     </header>
+
+    <section class="mini-profile">
+        <h1>Patrick Kenyon</h1>
+        <b>Personal tag-line</b>
```
Comment (mporam) [inline portfolio.html:20]: we use `<strong>` tags these days, `<b>` is old and deprecated

File: style.css [hunk 13]
```diff
@@ -1,5 +1,180 @@
...
+}
+
+.main-nav {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+}
+
+.mini-profile {
+    width: 100%;
```
Comment (mporam) [inline style.css:49]: this does nothing as block elements are 100% by default

File: style.css [hunk 14]
```diff
@@ -1,5 +1,180 @@
...
+    justify-content: center;
+    -webkit-box-flex: 1;
+    flex-shrink: 1;
+    -ms-flex-negative: 1;
+    flex-grow: 1;
+    -ms-flex-positive: 1;
+    flex-basis: 50%;
+    -ms-flex-preferred-size: 25%;
+    color: #fff;
+    background-image: url("https://placekitten.com/700/300");
```
Comment (mporam) [inline style.css:79]: YES!

File: style.css [hunk 15]
```diff
@@ -1,5 +1,180 @@
...
+    background-size: cover;
+    min-height: 300px;
+    background-position: center;
+    position: relative;
+    padding: 0 100px 0 100px;
```
Comment (mporam) [inline style.css:84]: this can be done with `0 100px`, no need to repeat it

File: style.css [hunk 16]
```diff
@@ -1,5 +1,180 @@
...
 
+.project:hover .project-over {
+    background: rgba(0,0,0,0.3);
+    height: 100%;
+    width: 100%;
+    color: rgba(212,175,55,1);
+    position: absolute;
+    top: 0;
+    left: 0;
+    padding: 0 100px 0 100px;
```
Comment (mporam) [inline style.css:95]: again, no repetition needed

**Review by mporam — CHANGES_REQUESTED** [review]

File: portfolio.html (hunk 12 above)
Comment (patrickkenyon) [reply portfolio.html:20]: Changed

File: style.css (hunk 13 above)
Comment (patrickkenyon) [reply style.css:49]: Removed

File: style.css (hunk 14 above)
Comment (patrickkenyon) [reply style.css:79]: :)

File: style.css (hunk 15 above)
Comment (patrickkenyon) [reply style.css:84]: Changed

File: style.css (hunk 16 above)
Comment (patrickkenyon) [reply style.css:95]: Changed

**Pull request merged**